src/r_tipping/RateSystem.jl

```diff
@@ -0,0 +1,98 @@
+# we consider the ODE dxₜ/dt = f(xₜ,λ(rt))
+# here λ = λ(t) ∈ Rᵐ is a function containing all the system parameters 
+
+# We ask the user to define: 
+#  1) a ContinuousTimeDynamicalSystem that should be investigated and
+#  2) a protocol for the time-dependent forcing with the struct RateProtocol
+
+# Then we give back the ContinuousTimeDynamicalSystem with the parameter 
+# changing according to the rate protocol
+"""
+    RateProtocol
+
+The RateProtocol contains all the fields (information) that allows to take an ODE of the form 
```

Comment:
If you want to be harmonious with the rest of DynamicalSystems.jl and DIfferentialEquations.jl, you need to replace λ with p throughout everything. The parameter container is always called `p` in both ecosystems.

I strongly recommend you do that because the choice of how to name this `λ` is completely arbitrary. So if there is any reason to prefer one option, you should do it. Here especially it is a very good reason.